packages/sandbox/src/app/sandbox/index.ts

```diff
@@ -99,6 +100,7 @@ export const sandboxItems: SandBoxSectionType[] = [
       { title: 'ListItemOption', SandBox: ListItemOptionSandbox },
       { title: 'ListItemValue', SandBox: ListItemValueSandbox },
       { title: 'ListItemGroup', SandBox: ListItemGroupSandbox },
+      { title: 'ListItemSelectable', SandBox: ListItemSelectableSandbox },
```

Comment:
Should we also add the support of `ListItemSelectable` into `ListItemGroup`? WDYT?

Reply:
Normally it's supported. The type is built on the same base than  `ListItemValueProps`

Reply:
Function `_isListItemSelectable_` created.